### PR TITLE
Add setScenarioState to module

### PR DESCRIPTION
**Pull request by mikhailsamarkin**
Added the ability to install scenario state

### CODE REVIEW

File: src/Module/Phiremock.php [hunk 1]
```diff
@@ -122,4 +123,10 @@ public function grabRequestsMadeToRemoteService(RequestBuilder $builder)
     {
         return $this->phiremock->listExecutions($builder);
     }
+
+    public function setScenarioState(string $name, string $state): void
```
Comment (mcustiel) [inline src/Module/Phiremock.php:127]: [change request] Please remove the type hints. Version 1 of this extension requires php5.6 as minimum.

**Comment by mcustiel** [comment]
Thank you for this PR @mikhailsamarkin I never noticed that was missing. Please check my comment in the PR, besides that everything looks great.

**Comment by mcustiel** [comment]
I already added the same feature in master of phiremock-codeception-module.

File: src/Module/Phiremock.php (hunk 1 above)
Comment (mikhailsamarkin) [reply src/Module/Phiremock.php]: I removed the type hints. Check again please.

P.S. This is my first PR to open-source projects ;)

**Comment by mikhailsamarkin** [comment]
> I already added the same feature in master of phiremock-codeception-module.

From `master` was removed `expectARequestToRemoteServiceWithAResponse` and another methods ;(

File: src/Module/Phiremock.php (hunk 1 above)
Comment (mcustiel) [reply src/Module/Phiremock.php]: Cool.
Thank you very much for your help and your PR.

**Comment by mcustiel** [comment]
> > I already added the same feature in master of phiremock-codeception-module.
> 
> From `master` was removed `expectARequestToRemoteServiceWithAResponse` and another methods ;(

Yes, in version 2 the module and the extension are 2 separated repositories. I added your change to master in the module: https://github.com/mcustiel/phiremock-codeception-module

**Review by mcustiel — COMMENTED** [review]
Approved

**Pull request merged**